lib/measured/unit.rb

```diff
@@ -56,8 +56,8 @@ def inverse_conversion_amount
 
   private
 
-  def case_insensitive(comparison)
-    [comparison].flatten.map(&:downcase)
+  def downcased_names
+    @downcased_names ||= @names.map(&:downcase)
```

Comment:
Ok I assumed this array would grow larger. In fact it contains a single unit's downcased aliases  right?